app/controllers/classifications.scala

```diff
@@ -0,0 +1,13 @@
+package controllers
+
+import javax.inject._
+import play.api.mvc._
+
+@Singleton
+class classifications @Inject()(cc: ControllerComponents) extends AbstractController(cc) {
```

Comment:
```suggestion
class Classifications @Inject()(cc: ControllerComponents) extends AbstractController(cc) {
```
Change casing for consistency